nwb_conversion_tools/gui/nwb_conversion_gui.py

```diff
@@ -649,15 +655,20 @@ def __init__(self, parent):
 
     def run(self):
         #try:
-        mod_file = self.parent.conversion_module_path
-        spec = importlib.util.spec_from_file_location(os.path.basename(mod_file).strip('.py'), mod_file)
-        conv_module = importlib.util.module_from_spec(spec)
-        spec.loader.exec_module(conv_module)
+        if len(self.parent.conversion_module_path)>0:# if not an empty string (if value was selected from gui)
+            mod_file = self.parent.conversion_module_path
```

Comment:
do not input conversion_module_path. Input the class itself!

Reply:
the class is being input, conversion_module path only exists if the user selects a path from the file drop down menu. If not, then it takes only the class into consideration